Source/UnrealEnginePython/Private/UEPyFARFilter.cpp

```diff
@@ -0,0 +1,322 @@
+#include "UnrealEnginePythonPrivatePCH.h"
+
+static PyObject *py_ue_farfilter_append(ue_PyFARFilter *self, PyObject * args) {
+	PyObject *pyfilter = nullptr;
+	py_ue_sync_farfilter((PyObject *)self);
+	if (!PyArg_ParseTuple(args, "|O:append", &pyfilter))
+		return NULL;
+	ue_PyFARFilter *filter = py_ue_is_farfilter(pyfilter);
+	self->filter.Append(filter->filter);
+	Py_RETURN_NONE;
+}
+
+static PyObject *py_ue_farfilter_clear(ue_PyFARFilter *self) {
+	py_ue_clear_farfilter(self);
+	self->filter.Clear();
+	Py_RETURN_NONE;
+}
+
+static PyObject *py_ue_farfilter_is_empty(ue_PyFARFilter *self, PyObject * args) {
+	py_ue_sync_farfilter((PyObject *)self);
+	if (self->filter.IsEmpty())
+		Py_RETURN_TRUE;
+	Py_RETURN_FALSE;
+}
+
+static PyMethodDef ue_PyFARFilter_methods[] = {
+	{ "append", (PyCFunction)py_ue_farfilter_append, METH_VARARGS, "" },
+	{ "clear", (PyCFunction)py_ue_farfilter_clear, METH_VARARGS, "" },
+	{ "is_empty", (PyCFunction)py_ue_farfilter_is_empty, METH_VARARGS, "" },
+	{ NULL }  /* Sentinel */
+};
+
+static PyObject *py_ue_farfilter_get_iooda(ue_PyFARFilter *self, void *closure) {
```

Comment:
better to use explicit names instead of acronysms,, it is boring but more 'pythonic' :)